src/components/SatelliteManifestPanel/__tests__/SatelliteManifestPanel.test.tsx

```diff
@@ -183,51 +217,42 @@ describe('Satellite Manifest Panel', () => {
     );
 
     fireEvent.click(getByTestId('expand-details-button-0'));
-    expect(container).toMatchSnapshot();
+    expect(container).toBeInTheDocument();
```

Comment:
What is this actually testing? If you render something to the dom and then check its in the dom this will always pass, but doesn't really give us any information as to what is being tested.